sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/batch_job.py

```diff
@@ -0,0 +1,59 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing import Dict
+
+from azure.ai.ml._schema._deployment.batch.batch_job import BatchJobSchema
+from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY
+from azure.ai.ml.entities._deployment.batch_job_property import BatchJobProperty
+from azure.ai.ml.entities import SystemData
+from azure.ai.ml._restclient.v2020_09_01_dataplanepreview.models import BatchJobResource as RestBatchJobResource
+
+
+class BatchJob:
+    """Batch Job entity
+
+    :param id: Fully qualified resource ID for the resource. Ex -
+     /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}.
+    :type id: str
+    :param name: The name of the resource.
+    :type name: str
+    :param type: The type of the resource. E.g. "Microsoft.Compute/virtualMachines" or
+     "Microsoft.Storage/storageAccounts".
+    :type type: str
+    :param properties: Required. [Required] Additional attributes of the entity.
+    :type properties: ~azure.mgmt.machinelearningservices.models.BatchJob
+    :param system_data: System data associated with resource provider.
+    :type system_data: SystemData
+    """
+
+    def __init__(
+        self,
+        id: str = None,
+        name: str = None,
+        type: str = None,
+        properties: BatchJobProperty = None,
```

Comment:
Sorry if I was not clear on this earlier. Can we promote fields of BatchJobProperty to top level as properties of BatchJob?

Reply:
Ah okay, I see what you mean now. Will make that fix